src/components/include/application_manager/application_manager.h

```diff
@@ -483,6 +489,13 @@ class ApplicationManager {
   GetCloudAppConnectionStatus(ApplicationConstSharedPtr app) const = 0;
 
   virtual bool IsHMICooperating() const = 0;
+
+  /*
+   * @brief Sets state for hmi cooperating
+   * @param New state to be set
```

Comment:
@ydementieiev For doxygen documentation you should specified a parameter name 
(e.g. `@param hmi_cooperating New state to be set`)

Reply:
@IGapchuk  done [faf31b](https://github.com/smartdevicelink/sdl_core/pull/3269/commits/faf31b567f45e2c8d77bec815805113a0b34ba63)